es/appendices/glossary.rst

```diff
@@ -49,22 +52,22 @@ Glossary
             )
 
     CSRF
-        Cross Site Request Forgery. Prevents replay attacks, double
-        submissions and forged requests from other domains.
+        Cross Site Request Forgery(falsificación de petición en sitios cruzados).
+        Previene ataques de reproducción,doble envío y peticiones falsas de otros dominios.
 
     routes.php
-        A file in APP/Config that contains routing configuration.
-        This file is included before each request is processed.
-        It should connect all the routes your application needs so
-        requests can be routed to the correct controller + action.
+        Un archivo en APP/Config que contiene configuración de enrutamiento.
+        Este archivo es incluido antes de que cualquier petición se procese.
+        Debe conectar todas las rutas que tu aplicación necesita para que las peticiones puedan ser
+        enrutadas al controlador + acción que corresponda.
 
     DRY
-        Don't repeat yourself. Is a principle of software development aimed at
-        reducing repetition of information of all kinds. In CakePHP DRY is used
-        to allow you to code things once and re-use them across your
-        application.
+        Don't repeat yourself(No te repitas a ti mismo). Es un principio del desarrollo de software
+        apuntado a reducir la repetición de información de cualquier tipo.
+        En CakePHP DRY es usado para permitirte programar to código una vez, y re-usarlo en
+        distintas partes de tu aplicación.
 
 
 .. meta::
-    :title lang=es: Glossary
-    :keywords lang=es: html attributes,array class,array controller,glossary glossary,target blank,dot notation,routing configuration,forgery,replay,router,syntax,config,submissions
+:title lang=es: Glosario
```

Comment:
What should i change there?
Should i keep the title in english? or its about indentation  in `:title lang=es: Glosario` ?
Thanks